x/concentrated-liquidity/swaps_test.go

```diff
@@ -2331,28 +2337,81 @@ func (s *KeeperTestSuite) TestCalcOutAmtGivenInWriteCtx() {
 			s.Require().Equal(poolBeforeCalc.GetLiquidity(), poolAfterCalc.GetLiquidity())
 			s.Require().Equal(poolBeforeCalc.GetTickSpacing(), poolAfterCalc.GetTickSpacing())
 
+			// check that fee accum has been correctly updated upon writeCtx
 			feeAccum, err := s.App.ConcentratedLiquidityKeeper.GetFeeAccumulator(s.Ctx, 1)
 			s.Require().NoError(err)
 
 			feeAccumValue := feeAccum.GetValue()
-			s.Require().Equal(0, feeAccumValue.Len())
-			s.Require().Equal(1,
+			s.Require().Equal(1, feeAccumValue.Len())
+			s.Require().Equal(0,
```

Comment:
I don't really get why this changed

Reply:
Previously, this was a non-mutative method. As a result, we expected no accumulator updates. It would only mutate upon `writeCtx` being called. Now, it is mutative so the update is expected immediately after the call